.github/workflows/test_full.yml

```diff
@@ -40,5 +40,10 @@ jobs:
         shell: bash
 
       - name: build coverage file
-        run: pytest --tb=long -vv --cache-clear --cov=dff tests/
```

Comment:
Don't you think it would be better to split this step into two and make use of conditional step execution?